package.json

```
@@ -18,9 +19,10 @@
     "generate": "stencil generate"
   },
   "dependencies": {
+    "@saasquatch/dom-context-hooks": "^0.1.0",
     "@stencil/core": "^1.17.4",
     "debug": "^4.3.1",
-    "dom-context": "^1.0.0",
+    "dom-context": "^1.0.1",
```

```suggestion
    "dom-context": "^1.2.0",
```
this version of dom-context suppresses the error spam